src/app/frontend/common/components/list/column/menu/component.ts

```diff
@@ -63,6 +64,10 @@ export class MenuComponent implements ActionColumn {
     this.typeMeta = typeMeta;
   }
 
+  setIsScalable(isScalable: boolean): void {
```

Comment:
s/IsScalable/scalable as mentioned above.

We can use setter/getter methods here.

```typescript
set scalable(...)

get scalable(): boolean
```

Usage will be more straightforward then.

Reply:
We don't need this anymore, because we're embedding the value in `TypeMeta`.